src/cli/backup/exchange.go

```diff
@@ -502,13 +503,14 @@ func runDetailsExchangeCmd(
 		return nil, errs.Fail(err)
 	}
 
-	d, _, err := r.BackupDetails(ctx, backupID)
-	if err != nil {
-		if errors.Is(err, kopia.ErrNotFound) {
+	d, _, errs := r.BackupDetails(ctx, backupID)
+	// TODO: log/track recoverable errors
+	if errs.Err() != nil {
+		if errors.Is(errs.Err(), kopia.ErrNotFound) {
```

Comment:
Do we need to add Clues here?

Reply:
Good question, and no.  The general handling is that if were passing a `ctx` into an internal func, then we should assume _that_ func handles packing clues into the error.  There's no harm if we over-pack the data.  But this helps minimize boilerplate.